common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html

```diff
@@ -37,7 +37,7 @@ <h3 class="commercial__title">@container.displayName</h3>
                         <div class="commercial__body">
                             @for(layout <- container.containerLayout) {
                                 @for(slice <- layout.slices) {
-                                    <ul class="lineitems l-row l-row--cols-4">
+                                    <ul class="lineitems l-row l-row--cols-@math.min(slice.columns.flatMap(_.cards).map(_.item).filter(_.isInstanceOf[ContentCard]).length, 4)">
```

Comment:
Perhaps move this expression into https://github.com/guardian/frontend/blob/master/common/app/views/support/Commercial.scala#L83?